third_party/proton/proton/viewer.py

```diff
@@ -187,6 +187,11 @@ def parse(metrics, filename, include, exclude, threshold, depth, format):
         # TODO: generalize to support multiple metrics, not just the first one
         gf = filter_frames(gf, include, exclude, threshold, metrics[0])
         print(gf.tree(metric_column=metrics, expand_name=True, depth=depth, render_header=False))
+        if (f"bytes (inc)" in metrics):
```

Comment:
1. It could be just "bytes (inc)" right?
2. No `(` and `)`.
3. Can you move the new code into a function? Maybe in the future we could have warnings for other metrics as well

Reply:
I didn't get that 2 was but I don't think we should remove the `( `and `)` around the entire if statement.